R/functions-XChromatogram.R

```diff
@@ -203,15 +203,17 @@ XChromatogram <- function(rtime = numeric(), intensity = numeric(),
 #' @param pks chromatographic peaks as returned by `chromPeaks(x)`.
 #'
 #' @noRd
-.add_chromatogram_peaks <- function(x, pks, col, bg, type, pch, ...) {
+.add_chromatogram_peaks <- function(x, pks, col, bg, type, pch,
+                                    yoffset = 0, ...) {
     switch(type,
            point = {
-               points(pks[, "rt"], pks[, "maxo"], pch = pch, col = col,
-                      bg = bg, ...)
+               points(pks[, "rt"], pks[, "maxo"] + yoffset, pch = pch,
+                      col = col, bg = bg, ...)
            },
            rectangle = {
                rect(xleft = pks[, "rtmin"], xright = pks[, "rtmax"],
-                    ybottom = rep(0, nrow(pks)), ytop = pks[, "maxo"],
+                    ybottom = rep(yoffset, nrow(pks)),
+                    ytop = pks[, "maxo"] + yoffset,
```

Comment:
this functions highlights identified chromatographic peaks in an EIC. The `"maxo"` is hardcoded because in most cases either the apex position (`type = "point"`) or a rectangle or polygon is drawn that has this apex position as its *highest* point. 
If you suggest to add an additional parameter that would allow the user to choose this, that would be tricky IMHO, because it would only work for `type = "point"` and `type = "rectangle"` but not for the other type(s).

Reply:
indeed, so leave as-is.